idunn/api/search.py

```diff
@@ -24,8 +25,14 @@ async def search(
     """
     # Fetch autocomplete results which will be filtered
     query.q = normalize(query.q)
+    if query.q == "":
+        return Response(status_code=200, content=build_empty_query_content_response())
     response = await get_autocomplete(query, extra)
 
+    # When no result was found for an acceptable query
+    if not response.features and query.q != "":
```

Comment:
Here `query.q` will never be empty, otherwise we would have early-returned a few lines above.

I have mixed feelings about this, so if this is intentional for the sake of explicitness I'm ok with it. Otherwise a comment would probably do the work :)